### PR TITLE
RUM-1837 Update telemetry schema

### PR DESCRIPTION
**Pull request by maciejburda**
Adds missing fields to configuration telemetry model

### CODE REVIEW

File: schemas/telemetry/configuration-schema.json [hunk 1]
```diff
@@ -288,6 +288,14 @@
                   "type": "integer",
                   "description": "The upload frequency of batches (in milliseconds)"
                 },
+                "batch_processing_level": {
+                  "type": "integer",
+                  "description": "Maximum number of batches processed sequencially without a delay"
+                },
+                "background_tasks_enabled": {
+                  "type": "boolean",
+                  "description": "Whether UIApplication background tasks are enabled"
```
Comment (0xnm) [inline schemas/telemetry/configuration-schema.json:297]: this needs to be changed, it is not only about `UIApplication` :)

Comment (maciejburda) [reply schemas/telemetry/configuration-schema.json:297]: I think it's a configuration only available on iOS 🤔

Comment (mariusc83) [reply schemas/telemetry/configuration-schema.json:297]: yeah, I was about to ask what's that property all about ?

**Review by mariusc83 — DISMISSED** [review]

File: schemas/telemetry/configuration-schema.json (hunk 1 above)
Comment (maciejburda) [reply schemas/telemetry/configuration-schema.json:297]: This is something that we forgot to add the other day and I discovered it when working on `batch_processing_level`

I can move it to a separate PR too :)

**Review by mariusc83 — APPROVED** [review]

**Pull request merged**